### PR TITLE
Meson more

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -1,11 +1,12 @@
 project('UEFITool', ['c', 'cpp'],
   version: 'A60',
```
Comment (NikolajSchlej) [inline meson.build]: A61

Comment (hughsie) [reply meson.build]: Fixed.

**Comment by NikolajSchlej** [comment]
LGTM, but we need to do something with that "version" field:
- if it's optional, I propose to just leave it unset for now
- if not, we need to change version.h format into something that doesn't require sed/awk to preprocess it, as we currently do to name releases.

**Comment by NikolajSchlej** [comment]
I'll open a separate issue on that version thing, no need to stall this PR for it.

**Comment by hughsie** [comment]
> if it's optional

It's kinda optional, etc:

    Distribution package /home/hughsie/Code/UEFITool/build/meson-dist/UEFITool-undefined.tar.xz tested
    Created /home/hughsie/Code/UEFITool/build/meson-dist/UEFITool-undefined.tar.xz

Do any of these seem acceptable? https://stackoverflow.com/questions/59201214/can-the-meson-project-version-be-assigned-dynamically

**Pull request merged**